pkg/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -234,6 +235,9 @@ type ClusterDeploymentStatus struct {
 	// +optional
 	CertificateBundles []CertificateBundleStatus `json:"certificateBundles,omitempty"`
 
+	// InstallStartedTimestamp is the time when all pre-requisites were met and cluster installation was launched.
+	InstallStartedTimestamp *metav1.Time `json:"installStartedTimestamp,omitempty"`
```

Comment:
s/Timestamp/Time per k8s API conventions.

Reply:
We unfortunately already used InstalledTimestamp in Hive before we learned this, though we are using Time elsewhere for other fields. Would we prefer InstalledTimestamp (which we can't change due to v1) + InstallStartedTime, or InstallStartedTimestamp bearing that discrepancy in mind? I'm ok with either.

Reply:
Gotcha, either seems reasonable then. It's a hive API, so I'm happy to defer to your team's preference.

Reply:
With it's proximity/similarity to the pre-existing InstalledTimestamp, it probably looks a little cleaner to do it wrong again and use Timestamp again within that CRD. I'll leave a TODO to correct them both if we ever go to a V2.